### PR TITLE
Remove unit title

### PR DESCRIPTION
**Pull request by wilkus27**
Issue: https://github.com/openstax/cnx-recipes/issues/1427

### CODE REVIEW

**Comment by wilkus27** [comment]
@helenemccarron @tomjw64 I was trying to trash unit title by recipes, in different passes, but each time I got such error:

image (11)

so I decided to use `display: none` property in styles. Please let me know what do you think about such solution.

File: styles/designs/carnival/parts/_unit-components.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    _properties: (
+
+    )
+);
+
+$UnitHeader: (
+    _name: "Header",
+    _subselector: " > h1",
+    _properties: (
+        display: none,
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_unit-components.scss]: I am bit on the fence on this one but I wonder if we should delete this kind of title in the recipes or hide it. This should not stop moving forward in this case, but I thought I'd at least ask the question for future reference. cc @brittany-johnson and @tomjw64

File: styles/books/anatomy/book.scss [hunk 2]
```diff
@@ -94,6 +94,7 @@
 
 @include use('BookRoot', "common:::BookRoot");
 @include use('Preface','carnival:::UnitPreface');
+@include use('Unit', 'carnival:::UnitShape');
```
Comment (tomjw64) [inline styles/books/anatomy/book.scss]: Since all that is happening is hiding, consider using `common:::Hidden`

**Comment by tomjw64** [comment]
@wilkus27 Your error in easybake seems strange. Do you have a branch?

**Comment by wilkus27** [comment]
@tomjw64 I use `common:::Hidden` as you requested. It's ready for review. Also I created a PR with trashing unit titles by recipes. Here is the link https://github.com/openstax/cnx-recipes/pull/1442

**Pull request merged**